app/models/qa/iri_template/variable_map.rb

```diff
@@ -0,0 +1,79 @@
+# Provide access to iri template variable map configuration.
+module Qa
+  module IriTemplate
+    class VariableMap
+      TYPE = "IriTemplateMapping".freeze
+      attr_reader :variable
```

Comment:
@elrayle I liked the comments in the class above in this spot.  I do not mind them below either.  For your future self I would make the place consistent

Reply:
My intention is for all class/module level doc to be the first line of the file.  I'll look to see if I put it elsewhere in another file.